samples/Network/enforce-firewall-route-table-on-subnet/README.md

```diff
@@ -0,0 +1,29 @@
+# Enforce Azure Firewall Route Table on every subnet
```

Comment:
I don't think the README or the display names/descriptions in the policy should refer to Azure Firewall.  As you mention in the readme there is nothing that forces the route table to target a firewall and I don't want people to get the wrong impression about what the policy is actually enforcing.

You could keep it in there as an example of why you might want to use a route table policy.